### DISCUSSION TITLE
Do you have any plans to open source this awesome integration builder features "split screen, auto scrolling, code highlighting, tab switching"?

### DISCUSSION THREAD

**Discussion by MadhuPolu** [markdoc/markdoc#53] [Q&A] [answered]
https://stripe.com/docs/payments/quickstart 

do you have any plans to offer this as built in ready to use configurable components through markdoc?

<img width="1678" alt="Screen Shot 2022-05-27 at 12 48 12 AM" src="https://user-images.githubusercontent.com/45701138/170561009-49134795-e8e4-4677-a4e9-f4e93d71b91b.png">

<img width="1415" alt="Screen Shot 2022-05-27 at 12 49 46 AM" src="https://user-images.githubusercontent.com/45701138/170561239-c368acdb-7b14-4884-9870-3a5aa11dec3d.png">

**Comment by mfix-stripe** [comment, marked as answer]
@MadhuPolu we do not have plans at the moment, but we will update here if that changes. Thanks for jumping into the Markdoc community! 💫
